doozerlib/distgit.py

```diff
@@ -1876,10 +1877,15 @@ def _update_csv(self, version, release):
                 raise DoozerFatalError('Error loading image-references data for {}'.format(self.metadata.distgit_key))
 
             try:
-                if name == self.metadata.image_name_short:  # ref is current image
+                # ref is current image
+                if name == self.metadata.image_name_short or name_in_bundle and ref['name'] == name_in_bundle:
                     image_tag = '{}:{}-{}'.format(name, version, release)
```

Comment:
This branch is currently broken (`name` always starts with `openshift/ose-` so this condition is always false) so we are never aware that `image_tag = '{}:{}-{}'.format(name, version, release)` is incorrect. The image name should be retrieved from `image_metadata.image_name_short` rather than hardcoded `openshift/ose-{tag_name}`.

Reply:
I suggest deleting this branch completely.